products/windows.md

```diff
@@ -9,236 +9,243 @@ releasePolicyLink: https://learn.microsoft.com/lifecycle/products/?terms=Windows
 activeSupportColumn: true
 releaseColumn: true
 releaseDateColumn: true
-releaseLabel: "Windows __RELEASE_CYCLE__"
 
 identifiers:
 -   cpe: cpe:2.3:o:microsoft:windows
 -   cpe: cpe:/o:microsoft:windows
 
 releases:
--   releaseCycle: "10, version 22H2 (E)"
+-   releaseCycle: "10-22H2"
+    releaseLabel: "10 22H2"
     releaseDate: 2022-10-18
     support: 2025-10-14
     eol: 2025-10-14
     latest: 10.0.19045
     link: https://learn.microsoft.com/windows/release-health/release-information
 
--   releaseCycle: "10, version 22H2 (W)"
-    releaseDate: 2022-10-18
-    support: 2025-10-14
-    eol: 2025-10-14
-    latest: 10.0.19045
-    link: https://learn.microsoft.com/windows/release-health/release-information
-
--   releaseCycle: "11, version 22H2 (E)"
+-   releaseCycle: "11-22H2-E"
+    releaseLabel: "11 22H2 (E)"
     releaseDate: 2022-09-20
     support: 2025-10-14
     eol: 2025-10-14
     latest: 10.0.22621
     link: https://learn.microsoft.com/windows/release-health/windows11-release-information
 
--   releaseCycle: "11, version 22H2 (W)"
+-   releaseCycle: "11-22H2-W"
+    releaseLabel: "11 22H2 (W)"
     releaseDate: 2022-09-20
     support: 2024-10-08
     eol: 2024-10-08
     latest: 10.0.22621
     link: https://learn.microsoft.com/windows/release-health/windows11-release-information
 
--   releaseCycle: "10, version 21H2 IoT"
+-   releaseCycle: "10-21H2-IoT"
+    releaseLabel: "10 21H2 IoT"
     releaseDate: 2021-11-16
+    lts: true
     support: 2027-01-12
     eol: 2032-01-13
-    lts: true
     latest: 10.0.19044
     link: https://learn.microsoft.com/windows/release-health/release-information
 
--   releaseCycle: "10, version 21H2"
+-   releaseCycle: "10-21H2"
+    releaseLabel: "10 21H2"
```

Comment:
Looks odd to have 10 21H2, 10 21H2 (E) and 10 21H2 (W)

Reply:
Resolved with 0dd496093cc1b5fe9b95d42f92732907e62e6c8e